src/page-modules/contact/ticket-control/feedback/feedbackForm.tsx

```diff
@@ -0,0 +1,390 @@
+import { FormEventHandler } from 'react';
+import { Button } from '@atb/components/button';
+import { SectionCard } from '../../components/section-card';
+import { ComponentText, PageText, useTranslation } from '@atb/translations';
+import { useLines } from '../../lines/use-lines';
+import { TransportModeType } from '@atb-as/config-specs';
+import { Input } from '../../components/input';
+import style from '../ticket-control.module.css';
+import { formMachine } from './feedbackformMachine';
+import { useMachine } from '@xstate/react';
+import { TicketData } from '../../server/types';
+import { andIf } from '@atb/utils/css';
+import {
+  ContextFeedbackForm,
+  fromContextFeedbackFormToTicketData,
+} from './utils';
+import { Line } from '../..';
+
+export const FeedbackForm = () => {
+  const { t } = useTranslation();
+  const { lines, getLinesByMode, getQuaysByLine } = useLines();
+  const [state, send] = useMachine(formMachine, {
+    input: {
+      subject:
+        PageText.Contact.ticketControl.title.no +
+        ' - ' +
+        PageText.Contact.ticketControl.subPageTitles.feedback.no,
+      date: new Date().toISOString().split('T')[0],
+      time: `${String(new Date().getHours()).padStart(2, '0')}:${String(new Date().getMinutes()).padStart(2, '0')}`,
+    },
+  });
+
+  const onSubmit: FormEventHandler<HTMLFormElement> = async (e) => {
+    e.preventDefault();
+
+    if (!state.matches('submitting')) return;
+
+    const { user, subject, description } = fromContextFeedbackFormToTicketData(
+      state.context as ContextFeedbackForm,
+    );
+
+    const ticketData: TicketData = {
+      subject: subject,
+      description: description,
+      user: user,
+    };
```

Comment:
I think that the `fromContextFeedbackFormToTicketData` should return an object of the type `TicketData` since its name `ToTicketData`